### ISSUE TITLE
Automatic Adoption for Access Point?

### ISSUE THREAD

**Issue by miquelfire** [mjp66/Ubiquiti#7]
I think if someone just bought the Access Point, it gets auto adopted before you see the screen in the guide. May want to put a note about that as I didn't need to adopt my access point when I got to that step. 

When I add a controller that can run all the time, I might have to adopt it on there though.

**Comment by mjp66** [comment]
I'll need more details to look at this. Step number, text, figure numbers, etc...
Also the date (revision) from the footer, so I know which doc you used.

**Comment by miquelfire** [comment]
12/30/2017 Step 61

Somehow, I didn't see Figure 124, it just went straight Figure 127, then step 62, when I hit next on Figure 123, and the AP had a status of connected. Figure 131 already had 1 WLAN when I first got there. Never saw Figure 133, but 137 when I started step 63. Still had to manually update the firmware.

My guess, there's a special state for these APs when they come from the factory (or at least the version of the factory firmware did this, but that was removed in a later version)
